package.json

```diff
@@ -56,8 +56,8 @@
     "coverage": "CI=true yarn test --coverage || true",
     "md-docs": "find ./src -name \\*.js | xargs npx jsdoc2md --template ./README.hbs --files | cat > ./FE_DOC.md",
     "html-docs": "npx jsdoc -c ./docs/config.json",
-    "madge": "npx madge ./src -i madge.png",
```

Comment:
Using madge library to generate graphs of module dependencies https://github.com/pahen/madge